sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc

```diff
@@ -231,6 +266,13 @@ Uses *BFloat16* in 3.X, Cooperative Matrix Operands +
 Uses *OpCooperativeMatrixPrefetchINTEL* instructions. +
  +
 | *{main_capability_name}* +
+| {capability_checked_token} | *{capability_checked_name}* +
+ +
+Uses *OpCooperativeMatrixLoadCheckedINTEL* and *OpCooperativeMatrixStoreCheckedINTEL*
+instructions. +
+ +
+| *{main_capability_name}* +
+
```

Comment:
I think we're missing a description of CooperativeMatrixOffsetInstructionsINTEL here.